packages/api-axios/src/api.js

```diff
@@ -287,7 +287,9 @@ export default class AvApi {
     const response = await this.query(config);
 
     const key = this.getQueryResultKey(response.data);
-    const totalPages = Math.ceil(response.data.totalCount / response.data.limit);
+    const length = response.data?.length;
```

Comment:
What if `response.data` is an object or string?

Reply:
existing functionality assumes its always an object or `getQueryResultsKey` wouldn't return a valid key